install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -413,28 +413,8 @@ mountOptions:
 
 The user can now create a PVC that uses this storage class.
 
-[[azure-unmanaged-disk]]
-=== Azure Unmanaged Disk Object Definition
-
-.azure-unmanaged-disk-storageclass.yaml
-[source,yaml]
-----
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: slow
-provisioner: kubernetes.io/azure-disk
-parameters:
-  skuName: Standard_LRS  <1>
-  location: eastus  <2>
-  storageAccount: azure_storage_account_name  <3>
-----
-<1> Azure storage account SKU tier. Default is empty.
-<2> Azure storage account location. Default is empty.
-<3> Azure storage account name. This must reside in the same resource group as the cluster. If a storage account is specified, the `location` is ignored. If a storage account is not specified, a new storage account gets created in the same resource group as the cluster.
-
 [[azure-advanced-disk]]
-=== Advanced Azure Disk Object Definition
+=== Azure Disk Object Definition
```

Comment:
@gaurav-nelson Sentence case?

Azure Disk object definition

(I think Disk is caps?)